website/docs/Contribute.md

```diff
@@ -155,23 +155,34 @@ Tests are automatically run via GitHub actions. There are two workflows:
 
 The first workflow is required to pass for all PRs (and it doesn't do any OpenAI calls). The second workflow is required for changes that affect the OpenAI tests (and does actually call LLM). The second workflow requires approval to run. When writing tests that require OpenAI calls, please use [`pytest.mark.skipif`](https://github.com/microsoft/autogen/blob/b1adac515931bf236ac59224269eeec683a162ba/test/oai/test_client.py#L19) to make them run in only when `openai` package is installed. If additional dependency for this test is required, install the dependency in the corresponding python version in [openai.yml](https://github.com/microsoft/autogen/blob/main/.github/workflows/openai.yml).
 
-#### Run non-OpenAI tests
+Make sure all tests pass, this is required for [build.yml](https://github.com/microsoft/autogen/blob/main/.github/workflows/build.yml) checks to pass
 
-To run the subset of the tests not depending on `openai` (and not calling LLMs)):
+#### Running tests locally
 
-- Install pytest:
+To run all tests, install the [test] option:
```

Comment:
What do you mean by "all" tests? Do they include the contrib tests? It's not enough to install [test] for running them.

Reply:
Not all tests, but if the dependencies in the contrib tests are not installed they will be skipped. You can see `build.yml` which only installs `pytest` and `mock` after the library. 

Maybe just say "run tests" ?

Reply:
I updated the doc with more information about contrib tests.